### PR TITLE
Adds go get kubevirt.io/containerized-data-importer support

### PR DESCRIPTION
**Pull request by copejon**
Enables `go get kubevirt.io/containerized-data-importer` by adding the respective meta tag and page.

### CODE REVIEW

File: pages/containerized-data-importer.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+layout: page
+title: 
+permalink: /kubevirt/
```
Comment (rmohr) [inline pages/containerized-data-importer.md]: I think the permalink needs to be different too.

Comment (copejon) [reply pages/containerized-data-importer.md]: Ah, okay.  As in `permalink: /containerized-data-importer/`?

**Comment by copejon** [comment]
@rmohr ptal

**Comment by rmohr** [comment]
Change is live and `go get` worked for me.

**Pull request merged**